storage/noxfile.py

```diff
@@ -1,18 +1,13 @@
 # pylint: disable=import-self,no-member
 import os
-
 import nox
 
 
 LOCAL_DEPS = ('../auth/', )
 
 
-@nox.session
```

Comment:
I'd much rather update to the new version of `nox` as a separate PR -- the less we can conflate separate changes, the faster it will be to review each piece. It would also be great to update all of the `gcloud-aio` subprojects' nox versions at once.

Reply:
Yes, please!  I've opened #49 to track this -- you can feel free to submit this as a separate PR, if you'd like (ie. just the nox changes), but don't feel like you need to! If not, I'll try to get to it sometime soon :)